cpan/Socket/Socket.xs

```diff
@@ -51,105 +51,6 @@
 # include <ws2tcpip.h>
 #endif
 
-#ifdef WIN32
```

Comment:
This file is maintained on CPAN, and cannot be edited here. Its bugtracker is here: https://rt.cpan.org/Public/Dist/Display.html?Name=Socket

Reply:
How can I send patch file there without creating another account?

Reply:
Yes, the last release was three days ago. You can [report bugs to RT via email](https://rt.cpan.org/NoAuth/RT/CPAN/Usage.html), or login to it using your github account (you need to click "Log out guest user" to access the login).

Reply:
Thanks, I shall try.

Reply:
Is there any way to clone that Socket module or I have to make the patch file from tarball?